android/app/src/main/java/com/zulipmobile/sharing/SharingHelper.kt

```diff
@@ -7,48 +7,25 @@ import android.content.ContentResolver
 import android.net.Uri
 import android.provider.OpenableColumns
 import android.util.Log
-import com.facebook.react.ReactApplication
 import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.ReactContext
 import com.facebook.react.bridge.WritableMap
 import com.zulipmobile.ZLog
 import com.zulipmobile.notifications.ReactAppStatus
 import com.zulipmobile.notifications.appStatus
 import com.zulipmobile.notifications.emit
-import com.zulipmobile.notifications.tryGetReactInstanceManager
 
 @JvmField
 val TAG = "ShareToZulip"
 
-/* Returns true just if we did handle the intent. */
-fun maybeHandleIntent(
```

Comment:
android notif [nfc]: Make maybeHandleIntent a helper method on MainActivity

nit: s/ notif//

Similarly for the next commit:

    android notif [nfc]: Simplify interface of MainActivity's maybeHandleIntent